external/fv3fit/fv3fit/emulation/layers/architecture.py

```diff
@@ -17,17 +17,25 @@ def __init__(
         """
         super().__init__(*args, **kwargs)
 
-        self.combine_axis = combine_axis
-        self.expand_axis = expand_axis
+        self._combine_axis = combine_axis
+        self._expand_axis = expand_axis
 
     def call(self, inputs):
 
-        if self.expand_axis is not None:
+        if self._expand_axis is not None:
             inputs = [
-                tf.expand_dims(tensor, axis=self.expand_axis) for tensor in inputs
+                tf.expand_dims(tensor, axis=self._expand_axis) for tensor in inputs
             ]
 
-        return tf.concat(inputs, axis=self.combine_axis)
+        return tf.concat(inputs, axis=self._combine_axis)
+
+    def get_config(self):
```

Comment:
not sure why these `get_config` methods are needed. Sounds like the call tracing approach is superior in terms of backwards compatibility since these classes won't need to be stable to load a saved model. did the ser/deser not work before?

The tensorflow [docs](https://www.tensorflow.org/tutorials/keras/save_and_load#saving_custom_objects) claim:
> If you are using the SavedModel format, you can skip this section. The key difference between HDF5 and SavedModel is that HDF5 uses object configs to save the model architecture, while SavedModel saves the execution graph. Thus, SavedModels are able to save custom objects like subclassed models and custom layers without requiring the original code.